RELEASING.md

```diff
@@ -8,11 +8,18 @@ This project does not have a strict release schedule. However, we would make a r
   - No release will be done if no changes got merged to the `master` branch during the above mentioned window.
   - Releases may be done more frequently than the above mentioned window.
 
-## Make Sure Everything Works
+## Prerelease checklist
 
+* Check and upgrade dependencies where it applies and makes sense.
+  - Create a distinct pull request and test your changes since it maybe introduce regressions.
+  - While using latest dependencies is advised it may not always be possible. 
```

Comment:
```suggestion
  - While using the latest versions of dependencies is advised, it may not always be possible due to potential compatibility issues. 
```